### PR TITLE
Added Udemy Resource Link

### LINKED ISSUE #224: Add Udemy Python Course to Resources
We should add [Udemy's Beginner's Guide to Python](https://blog.udemy.com/python-tutorial/) to the resources page

### PR DESCRIPTION
**Pull request by eunicekokor**
to close #224 

not sure why Travis isn't passing :disappointed:

### CODE REVIEW

File: data/resources.json [hunk 1]
```diff
@@ -306,6 +306,11 @@
           "url": "http://learn.adicu.com/python/",
           "description": "Written and developed by ADI, this tutorial covers basics of programming in Python.",
           "adi": true
+        },
+        {
+          "name": "Udemy Beginner's Guide to Python",
+          "url": "https://blog.udemy.com/python-tutorial/",
+          "description": "This is a guide developed for beginners by Udemy gives a short introduction to Python essentials and core concepts!"
```
Comment (thebrianzeng) [inline data/resources.json]: i think there's a grammar mistake here

**Comment by thebrianzeng** [comment]
looks like travis is failing because it's not getting the `key.lower()` change

**Comment by eunicekokor** [comment]
I think we need an extra `)`

**Comment by danrschlosser** [comment]
At your latest, commit, [this line](https://github.com/eunicekokor/adi-website/blob/5711071fe44cd41501beaaebed3eea29fa203be9/config/flask_config.py#L44) should have `key.lower()`, not `key`.

**Comment by eunicekokor** [comment]
hey! it was just a lil error w/ .format `("adi-website/{}").format(key)` instead of `"adi-website/{}".format(key)`. It seems like case wasn't a factor. I think this is ready to go!

**Comment by eunicekokor** [comment]
also @danrschlosser you definitely aren't a good person
you're a great person! :+1:

**Comment by thebrianzeng** [comment]
@eunicekokor we need `key.lower()` for it to work with the consul library (which is why we changed it before)

**Comment by eunicekokor** [comment]
why?

**Comment by danrschlosser** [comment]
The consul library requires all lowercase strings (this was an undocumented change, it seems).  Us passing things like `SECRET_KEY` used to work, but now we need to convert them to `secret_key`.

**Comment by eunicekokor** [comment]
Okay. Is that okay that it makes our tests fail?

On Tue, Aug 18, 2015 at 11:24 AM Dan Schlosser <notifications@github.com>
wrote:

> The consul library requires all lowercase strings (this was an
> undocumented change, it seems). Us passing things like SECRET_KEY used to
> work, but now we need to convert them to secret_key.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adicu/adi-website/pull/225#issuecomment-132307453>.
>
-- 
Eunice Kokor

**Pull request merged**